examples/esp32/CMakeLists.txt

```diff
@@ -6,3 +6,7 @@ cmake_minimum_required(VERSION 3.16)
 
 include($ENV{IDF_PATH}/tools/cmake/project.cmake)
 project(esp32-libmicrofido2)
+
+if (CONFIG_MBEDTLS_HARDWARE_AES STREQUAL "y")
```

Comment:
Probably this could now be removed, couldn't it?

Reply:
Indeed.